client/src/components/ratings_reviews/RatingsAndReviews.js

```diff
@@ -0,0 +1,64 @@
+import React, {component, useState, useEffect} from 'react';
```

Comment:
Purely out of curiosity: where and for what reason are you using the 'component' method that you are importing?

Reply:
honestly, i have not used component and i think we can get rid of it!